versions.props

```diff
@@ -29,7 +29,6 @@ io.prometheus:*=0.16.0
 io.swagger.core.v3:*=2.2.22
 jakarta.ws.rs:jakarta.ws.rs-api=3.1.0
 junit:junit=4.13.2
-net.sourceforge.argparse4j:argparse4j=0.9.0
```

Comment:
The corresponding `licenses` files can be removed now then too I think, opened #2627 for that.